lms/static/scripts/frontend_apps/config.ts

```diff
@@ -223,7 +223,12 @@ export type Product = {
 
 export type WeekDay = 'mon' | 'tue' | 'wed' | 'thu' | 'fri' | 'sat' | 'sun';
 
-export type EmailPreferences = Record<WeekDay, boolean>;
+export type SelectedDays = Record<WeekDay, boolean>;
+
+export type EmailPreferences = {
+  selectedDays: SelectedDays;
+  result: string | null;
```

Comment:
The meaning of "result" is a little unclear IMO. Perhaps this could be say `flashMessage` to match the backend terminology.

Reply:
Agreed